include/lsst/meas/base/PixelFlags.h

```diff
@@ -94,10 +79,13 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
     ) const;
 
 private:
-
+    typedef std::map<std::string, afw::table::Key<afw::table::Flag>> KeyMap;
+    typedef afw::image::MaskedImage<float> MaskedImageT;
     Control _ctrl;
-    FlagHandler _flagHandler;
-    SafeCentroidExtractor _centroidExtractor;
+    KeyMap _centerKeys;
+    KeyMap _anyKeys;
+    afw::table::Key<afw::table::Flag> _generalFailure;
+    afw::table::Key<afw::table::Flag> _offImageKey;
```

Comment:
For consistency, shouldn't it be `_generalFailureKey`? (or `_offImage`, with no `Key`, I guess).